pandas/core/arrays/datetimes.py

```diff
@@ -402,13 +504,15 @@ def _resolution(self):
     # Array-Like / EA-Interface Methods
 
     def __array__(self, dtype=None):
+        # TODO(datetime-tz __array__): push to parent
+        # If deprecating behavior for datetime-tz, we'll need to handle that
+        # specially.
         if is_object_dtype(dtype):
             return np.array(list(self), dtype=object)
         elif is_int64_dtype(dtype):
             return self.asi8
 
-        # TODO: warn that conversion may be lossy?
-        return self._data.view(np.ndarray)  # follow Index.__array__
+        return self._data
```

Comment:
don't we have another PR for this?

Reply:
Possibly https://github.com/pandas-dev/pandas/pull/23593/files?

Reply:
right ok